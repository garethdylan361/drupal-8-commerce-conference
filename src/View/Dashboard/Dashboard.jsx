class Dashboard extends React.Component {

  constructor(props){
    super(props);
  }

  handleSelect(eventKey) {
    event.preventDefault();
  }

  render(){

    let Row = ReactBootstrap.Row,
      Col = ReactBootstrap.Col;

    return(
      <content>
        <Row>

          <Col sm={3}>
            <Row>
              <Col sm={12}>
                <ConferenceSideNav />
              </Col>
            </Row>
          </Col>

          <Col sm={9}>
            <Row>
              <Col sm={12}>git
                <ConferenceNavBar />
              </Col>
            </Row>

            <Row>
              <Col sm={12}>
                Content
              </Col>
            </Row>

          </Col>

        </Row>

      </content>
    );
  }

}

ReactDOM.render(<Dashboard/>, document.getElementById("conferenceReactApp"));
