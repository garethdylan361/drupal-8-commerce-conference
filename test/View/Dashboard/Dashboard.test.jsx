import React from 'react';
import renderer from 'react-test-renderer';
import Dashboard from '../../../src/View/Dashboard/Dashboard';

test('Dashboard should render', () => {
  const dashboard = renderer.create(<Dashboard/>);
  expect(dashboard).toMatchSnapshot();
});
