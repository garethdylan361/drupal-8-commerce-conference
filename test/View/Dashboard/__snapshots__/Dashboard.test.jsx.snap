// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should render 1`] = `
<content>
  <div
    className="row"
  >
    <div
      className="col-lg-3 col-md-4 col-sm-5"
      xl={2}
    >
      <div
        className="row"
      >
        <div
          className="col-sm-12"
        >
          <ul
            className="nav nav-pills nav-stacked"
            role={null}
          >
            <li
              className="active"
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Conference 2017
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Conference 2016
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Conference 2015
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Conference 2014
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Conference 2013
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Conference 2012
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Conference 2011
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
    <div
      className="col-lg-9 col-md-8 col-sm-7"
      xl={10}
    >
      <div
        className="row"
      >
        <div
          className="col-sm-12"
        >
          <ul
            className="nav nav-tabs"
            role={null}
          >
            <li
              className="active"
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Analytics
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Manage
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Registration
              </a>
            </li>
            <li
              className=""
              role="presentation"
              style={undefined}
            >
              <a
                href="#"
                onClick={[Function]}
                onKeyDown={[Function]}
                role="button"
              >
                Name Tags
              </a>
            </li>
          </ul>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-sm-12"
        />
      </div>
    </div>
  </div>
</content>
`;
